smoketest/scripts/cli/test_load_balancing_inbound-ipvsadm.py

```diff
@@ -0,0 +1,149 @@
+                        self.assertIn(f'127.{backend_id}.0.2', config)
+                    self.assertIn(str(virtual_port), config)
+                    
+                    if mode == 'direct':
+                        self.assertIn('-g', config)
+                    elif mode == 'tunnel':
+                        self.assertIn('-i', config)
+                    elif mode == 'nat':
+                        self.assertIn('-m', config)
+                    
+                    if protocol == 'tcp':
+                        self.assertIn('-t', config)
+                    elif protocol == 'udp':
+                        self.assertIn('-u', config)
+                    elif protocol == 'sctp':
+                        self.assertIn('--sctp-service', config)
+    
+    def test_load_balancing_inbound_no_backend(self):
+        rule = 0
+        virtual_port = 8080
+        for protocol in ['tcp', 'udp', 'sctp']:
+            for mode in ['direct', 'tunnel', 'nat']:
+                for ip in [4, 6]:
+                    rule += 1
+                    virtual_port += 1
+                    if ip == 6:
+                        self.cli_set(base_path + ['rule', str(rule), 'virtual-address', virtual_ip6])
+                    elif ip == 4:
+                        self.cli_set(base_path + ['rule', str(rule), 'virtual-address', virtual_ip])
+                    self.cli_set(base_path + ['rule', str(rule), 'port', str(virtual_port)])
+                    self.cli_set(base_path + ['rule', str(rule), 'mode', mode])
+                    self.cli_set(base_path + ['rule', str(rule), 'protocol', protocol])
+        self.cli_commit()
+        
+        # Check configured port
+        config = read_file(IPVSADM_CONF)
+        
+        virtual_port = 8080
+        for protocol in ['tcp', 'udp', 'sctp']:
+            for mode in ['direct', 'tunnel', 'nat']:
+                for ip in [4, 6]:
+                    virtual_port += 1
+                    
+                    if ip == 6:
+                        self.assertIn(f'{virtual_ip6}', config)
+                    elif ip == 4:
+                        self.assertIn(f'{virtual_ip}', config)
+                    self.assertIn(str(virtual_port), config)
+                    
+                    if mode == 'direct':
+                        self.assertIn('-g', config)
+                    elif mode == 'tunnel':
+                        self.assertIn('-i', config)
+                    elif mode == 'nat':
+                        self.assertIn('-m', config)
+                    
+                    if protocol == 'tcp':
+                        self.assertIn('-t', config)
+                    elif protocol == 'udp':
+                        self.assertIn('-u', config)
+                    elif protocol == 'sctp':
+                        self.assertIn('--sctp-service', config)
+    
+    def test_load_balancing_inbound_no_rule(self):
+        self.cli_set(base_path)
+        self.cli_commit()
```

Comment:
Again, where is the validation?

Reply:
This does not require validation, which is to verify that it works when no rules are configured